src/rpc/net.cpp

```diff
@@ -212,17 +212,16 @@ static UniValue addnode(const JSONRPCRequest& request)
     std::string strCommand;
     if (!request.params[1].isNull())
         strCommand = request.params[1].get_str();
-    if (request.fHelp || request.params.size() != 2 ||
+    if (request.fHelp || request.params.size() < 2 || request.params.size() > 3 ||
         (strCommand != "onetry" && strCommand != "add" && strCommand != "remove"))
         throw std::runtime_error(
             RPCHelpMan{"addnode",
                 "\nAttempts to add or remove a node from the addnode list.\n"
-                "Or try a connection to a node once.\n"
-                "Nodes added using addnode (or -connect) are protected from DoS disconnection and are not required to be\n"
-                "full nodes/support SegWit as other outbound peers are (though such peers will not be synced from).\n",
+                "Or try a connection to a node once.\n",
                 {
                     {"node", RPCArg::Type::STR, /* opt */ false, /* default_val */ "", "The node (see getpeerinfo for nodes)"},
                     {"command", RPCArg::Type::STR, /* opt */ false, /* default_val */ "", "'add' to add a node to the list, 'remove' to remove a node from the list, 'onetry' to try a connection to the node once"},
+                    {"privileged", RPCArg::Type::BOOL, /* opt */ true, /* default_val */ "true", "If true, nodes added will be protected from DoS disconnection and not required to be full nodes or support segwit as other outbound peers are (though such peers will not be synced from). Only supported for command \"onetry\" for now."},
```

Comment:
```suggestion
                    {"privileged", RPCArg::Type::BOOL, /* default */ "true", "If true, nodes added will be protected from DoS disconnection and not required to be full nodes or support segwit as other outbound peers are (though such peers will not be synced from). Only supported for command \"onetry\" for now."},
```

Reply:
I don't see how that would compile.

Reply:
I understand your frustration you recently raised on IRC that pull requests frequently need to be rebased and that you prefer not to rebase unless the pull request is ready for merge. However, a pull request that needs rebase is less likely to receive review, since reviewers can't trivially run the latest tests on it (without having to solve a merge conflict). Also, they'd need to redo the solution of the merge conflict after you solve it to re-ACK, so that drives back reviewers even more.
I think solving trivial merge conflicts is a nice courtesy to reviewers. If the pull request in general does not attract reviewers that seems like a separate problem.